modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs using the form view
+
+.Prerequisites
+
+* You must have administrator privileges.
+
+[NOTE]
```

Comment:
@debsmita1 @sanketpathak @jerolimov - Guys I have added this little note after discussing it with my colleague @shipsing to provide more completeness to this procedure. We felt that adding this note in my procedure would ensure that the user is aware that a similar type of procedure works for adding quick starts, cluster roles, and pages in the Web Console. This would ensure that we don't miss anything with respect to content completeness. 

Kindly have a quick look at this minor addition and let me know your feedback, if any.

@Cbremble and @TanyaShearonRH - Tagging you guys here for reference.

Reply:
@Dhruv-Soni11, Please see my suggestions:
* Please add this note in the beginning of the .procedure section, not in the prerequistes section
* Please rephrase the note like: You can use a similar procedure to customize the following UI items:
** Quick starts visible to users
** Cluster roles accessible to a project
** Actions visible on the +Add page
* Suggest to make the last step generic: In the respective setting, enable or disable the options in the list, as needed.

Reply:
Fixed.

Reply:
Looks good to me